packages/core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -174,10 +173,6 @@ export default class Tooltip extends PureComponent<TooltipProps> {
     this.wasShown = false;
   }
 
-  getContainer() {
```

Comment:
It can be provided externally but no need to adjust it to tooltips-container here anymore